deebot_client/map.py

```diff
@@ -407,7 +408,10 @@ async def on_map_set(event: MapSetEvent) -> None:
 
         async def on_map_subset(event: MapSubsetEvent) -> None:
             if event.type == MapSetType.ROOMS and event.name:
-                room = Room(event.name, event.id, event.coordinates)
+                decompressed_coords = _decompress_7z_base64_data(event.coordinates).decode('utf-8')
```

Comment:
Compressed coordinates are Part of `GetMapSubset_V2` but not the `GetMapSubset` and therefore should be in the command class otherwise this code will break older bots